packages/sources/tradinghours/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@chainlink/tradinghours-adapter",
+  "version": "0.0.1",
+  "description": "Chainlink TradingHours adapter",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "tradinghours"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "1.3.0",
```

Comment:
Can we use the latest version?

Reply:
Yup, this uses the same version that all other EAs are currently using. I can make a separate PR to update all EAs.